CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
-## [Unreleased]
+## [1.12] - 2020-06-04
```

Comment:
You might want to create link for 1.12 at the bottom of this file... Also there is not need to remove [Unreleased]. We can keep it for later...